mongo/integration/collection_test.go

```diff
@@ -765,6 +765,18 @@ func TestCollection(t *testing.T) {
 			limit := limitVal.Int64()
 			assert.Equal(mt, int64(1), limit, "expected limit 1, got %v", limit)
 		})
+		mt.Run("maxTime", func(mt *mtest.T) {
```

Comment:
Instead of having a test specifically for this option, can we change the `options` test below to test all of the options at once?

Reply:
done.